src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarPublisher.java

```diff
@@ -98,6 +98,31 @@ public String getVisibilityFilter() {
         return this.visibilityFilter;
     }
 
+    // projectName may be a
```

Comment:
Not sure what this comment is intended to express:

    // projectName may be a

Reply:
Thanks. I'll remove it.